peachjam/adapters/adapters.py

```diff
@@ -109,8 +109,18 @@ def get_doc_list(self):
             url = f"{self.api_url}/akn/{place_code}/.json"
             while url:
                 res = self.client_get(url).json()
-                results.extend(res["results"])
-                url = res["next"]
+
+                # Filter by actor, if setting is present
+                actor = self.settings.get("actor", None)
```

Comment:
doesn't .get() return None by default if nothing is found?

Reply:
> doesn't .get() return None by default if nothing is found?

Yes, it does.